### ISSUE TITLE
Time

### ISSUE THREAD

**Issue by dycasxe101** [dorajam/Convolutional-Network#4]
How long do you spent ur time to training and testing?

**Comment by dorajam** [comment]
I would not use this code for training a model. The objective of this implementation is to understand the innerworkings of the network. If you're looking to use CNNs for your project, I'd recommend looking at a higher level library e.g. Tensorflow or Torch.

**Comment by dycasxe101** [comment]
My project used theano, but my lecturer doesn't allow to use any libraries. Behonest, iam so confuse now how to deal my project

**Comment by dorajam** [comment]
I have not trained this model, so I can't answer your question. But when running the model, it took a very long time, to start showing reasonable results for the weight kernels. My suggestion for you is to skip training the model, I'm sure your teacher wants to see that you understand the material and can implement it from scratch rather than having a fully-functional fast CNN. You won't get that with a python convolutional network.

If your homework though is to build a neural network in python and train it, I suggest you looking at this repo [here](https://github.com/dorajam/Neural-Nets/blob/master/networks/nnetwork.py). This should be ~30minutes to train if I remember correctly.

**Comment by dycasxe101** [comment]
In my major, we aren't allowed to used library for classification to fulfill essay. Btw the code almost done now in java, but i have problem about dropout.

**Comment by dycasxe101** [comment]
But for all, thanks @dorajam
